package.json

```diff
@@ -84,6 +84,7 @@
     "webdriverio": "^8.36.1"
   },
   "resolutions": {
+    "@chainsafe/ssz": "^0.19.1",
```

Comment:
why do we need this here?

Reply:
if lock file is correctly updated it should be fine

Reply:
there are `@ethereumjs` deps that use ssz@11.1.0 which will cause issues potentially.... is in `prover` so not the end of the world but probably not a bad things we are updating that one to use faster hashing and whatnot

Reply:
yeah but the other packages have the version the want defined in their package.json though, so should be fine